client/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html

```diff
@@ -14,38 +14,57 @@
                   novalidate
                   data-ng-submit="DefaultVoicemailCtrl.saveDefaultVoicemail()">
                 <div class="form-group">
-                    <label class="control-label"
-                           for="default"
-                           data-translate="telephony_line_answer_default_voicemail_label">
-                    </label>
-                    <select id="default"
-                            name="default"
-                            class="form-control"
-                            required
-                            data-ng-model="DefaultVoicemailCtrl.options.defaultVoicemail"
-                            data-ng-change="DefaultVoicemailCtrl.success = false"
-                            data-ng-options="number.serviceName as DefaultVoicemailCtrl.formatNumber(number) for number in DefaultVoicemailCtrl.numbers | orderBy: ['description', 'label', 'serviceName']">
-                    </select>
+                    <div class="row">
```

Comment:
`col-**-**` could be a direct child of `form-group`.
ref: https://getbootstrap.com/docs/3.3/css/#forms-horizontal